components/application-mgt/org.wso2.carbon.identity.application.mgt.ui/src/main/resources/web/application/configure-authentication-flow.jsp

```diff
@@ -488,11 +489,11 @@ var img = "";
 							      
 							      	       <tr>
 							      	      	   <td>
-							      	      		<input name="step_<%=step.getStepOrder()%>_fed_auth" id="" type="hidden" value="<%=idp.getIdentityProviderName()%>" />
-							      	      			<%=idp.getIdentityProviderName() + (idpEnableStatus.get(idp.getIdentityProviderName()) != null && idpEnableStatus.get(idp.getIdentityProviderName()) ? "" : disbleText) %>
+							      	      		<input name="step_<%=step.getStepOrder()%>_fed_auth" id="" type="hidden" value="<%=Encode.forHtmlAttribute(idp.getIdentityProviderName())%>" />
+							      	      			<%=Encode.forHtmlAttribute(idp.getIdentityProviderName()) + idpEnableStatus.get(Encode.forHtmlAttribute(idp.getIdentityProviderName())) != null && idpEnableStatus.get(Encode.forHtmlAttribute(idp.getIdentityProviderName())) ? "" : disbleText %>
```

Comment:
This should be as below.
<%=Encode.forHtmlContent(idp.getIdentityProviderName()) + (idpEnableStatus.get(idp.getIdentityProviderName()) != null && idpEnableStatus.get(idp.getIdentityProviderName()) ? "" : disbleText) %>